rules/data/embedded/embedded-svg.yara

```diff
@@ -0,0 +1,46 @@
+rule svg: harmless {
+  meta:
+    description = "Contains SVG (Scalable Vector Graphics) content"
+
+  strings:
+    $svg_open1  = "<svg>"
+    $svg_open2  = "<SVG>"
+    $svg_close1 = "</svg>"
+    $svg_close2 = "</SVG>"
+
+  condition:
+    all of them
```

Comment:
you probably want something like `all of ($upper*) or all of ($lower*)`

Reply:
Good catch -- updated in [`efa05ad` (#704)](https://github.com/chainguard-dev/malcontent/pull/704/commits/efa05ad402738bbeda438125be14027e84461b5d).